samples/client/petstore/javascript-promise/test/ApiClientTest.js

```diff
@@ -380,13 +380,14 @@ function makeDumbRequest(apiClient, opts) {
   var httpMethod = opts.httpMethod || 'GET';
   var pathParams = opts.pathParams || {};
   var queryParams = opts.queryParams || {};
+  var collectionQueryParams = opts.collectionQueryParams || {};
   var headerParams = opts.headerParams || {};
   var formParams = opts.formParams || {};
   var bodyParam = opts.bodyParam;
   var authNames = [];
   var contentTypes = opts.contentTypes || [];
   var accepts = opts.accepts || [];
   var callback = opts.callback;
-  return apiClient.callApi(path, httpMethod, pathParams, queryParams,
+  return apiClient.callApi(path, httpMethod, pathParams, queryParams, collectionQueryParams,
```

Comment:
No worry. I just want to play safe so that's why I suggest you file the change against 2.3.0 branch instead.